### PR TITLE
Use triton==2.0.0

### PR DESCRIPTION
**Pull request by jongwook**

### CODE REVIEW

**Comment by r5rinimaa** [comment]
there is no triton==2.0.0 in pip install option

**Comment by jongwook** [comment]
@r5rinimaa Hi, which OS / platform are you on? It lists triton as dependency only on x86_64 linux, for which PyPI provides a pre-built wheels: https://pypi.org/project/triton/2.0.0/#files

Happy to look if you are seeing any errors or stacktraces because of it!

**Comment by r5rinimaa** [comment]
hi,the OS is ubuntu20.0.4 PyTorch  1.11.0
Python  3.8(ubuntu20.04)
Cuda  11.3

**Comment by jongwook** [comment]
I think pip should work, unless you're on non-x86_64 such as ARM. Can you post the output of 

```
uname -a
```

and

```
pip install triton==2.0.0 --log log.txt
```

as well as what's saved in `log.txt` after that?

**Comment by r5rinimaa** [comment]
uname -a
Linux autodl-container-6a4811bc52-d71f00b6 5.4.0-126-generic #142-Ubuntu SMP Fri Aug 26 12:12:57 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux                                                                                                                                                                    
Skipping link: not a file: https://repo.huaweicloud.com/repository/pypi/simple/triton/
2023-03-10T09:41:09,461 Given no hashes to check 0 links for project 'triton': discarding no candidates
2023-03-10T09:41:09,462 ERROR: Could not find a version that satisfies the requirement triton==2.0.0 (from versions: 0.4.1, 0.4.2, 1.0.0, 1.1.0, 1.1.1, 2.0.0.dev20221030, 2.0.0.dev20221031, 2.0.0.dev20221101, 2.0.0.dev20221103, 2.0.0.dev20221105, 2.0.0.dev20221117, 2.0.0.dev20221120, 2.0.0.dev20221202, 2.0.0.dev20230208, 2.0.0.dev20230217, 2.0.0a1, 2.0.0a2)
2023-03-10T09:41:09,462 ERROR: No matching distribution found for triton==2.0.0
2023-03-10T09:41:09,462 Exception information:
2023-03-10T09:41:09,462 Traceback (most recent call last):
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_vendor/resolvelib/resolvers.py", line 341, in resolve
2023-03-10T09:41:09,462     name, crit = self._merge_into_criterion(r, parent=None)
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_vendor/resolvelib/resolvers.py", line 173, in _merge_into_criterion
2023-03-10T09:41:09,462     raise RequirementsConflicted(criterion)
2023-03-10T09:41:09,462 pip._vendor.resolvelib.resolvers.RequirementsConflicted: Requirements conflict: SpecifierRequirement('triton==2.0.0')
2023-03-10T09:41:09,462 
2023-03-10T09:41:09,462 During handling of the above exception, another exception occurred:
2023-03-10T09:41:09,462 
2023-03-10T09:41:09,462 Traceback (most recent call last):
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 127, in resolve
2023-03-10T09:41:09,462     result = self._result = resolver.resolve(
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_vendor/resolvelib/resolvers.py", line 473, in resolve
2023-03-10T09:41:09,462     state = resolution.resolve(requirements, max_rounds=max_rounds)
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_vendor/resolvelib/resolvers.py", line 343, in resolve
2023-03-10T09:41:09,462     raise ResolutionImpossible(e.criterion.information)
2023-03-10T09:41:09,462 pip._vendor.resolvelib.resolvers.ResolutionImpossible: [RequirementInformation(requirement=SpecifierRequirement('triton==2.0.0'), parent=None)]
2023-03-10T09:41:09,462 
2023-03-10T09:41:09,462 The above exception was the direct cause of the following exception:
2023-03-10T09:41:09,462 
2023-03-10T09:41:09,462 Traceback (most recent call last):
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_internal/cli/base_command.py", line 180, in _main
2023-03-10T09:41:09,462     status = self.run(options, args)
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_internal/cli/req_command.py", line 205, in wrapper
2023-03-10T09:41:09,462     return func(self, options, args)
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_internal/commands/install.py", line 318, in run
2023-03-10T09:41:09,462     requirement_set = resolver.resolve(
2023-03-10T09:41:09,462   File "/root/miniconda3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 136, in resolve
2023-03-10T09:41:09,462     raise error from e
2023-03-10T09:41:09,462 pip._internal.exceptions.DistributionNotFound: No matching distribution found for triton==2.0.0
2023-03-10T09:41:09,466 Removed build tracker: '/tmp/pip-req-tracker-dl31st9p'

**Comment by jongwook** [comment]
It appears that your PIP setting (or your firewall) is preventing you from directly accessing PyPI where triton 2.0.0 is stored.

See if you can directly download the wheel from this URL: https://files.pythonhosted.org/packages/57/b6/d93645e54a7a7ceba5c8a94cfc843f2c17b47853fc106527afc5562a40e8/triton-2.0.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl

after which you can do:

```bash
pip install triton-2.0.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

File: setup.py
Comment (htownbeautycartel-tech) [inline setup.py]: @ @

>

**Pull request merged**